plugins/webp-uploads/helper.php

```diff
@@ -175,6 +172,45 @@ function webp_uploads_generate_additional_image_source( $attachment_id, $image_s
 	);
 }
 
+/**
+ * Generates a file name for an image with a different extension.
+ *
+ * @param WP_Image_Editor $editor    The image editor that handle the image manipulation.
+ * @param string          $file      Full path to the image file.
+ * @param string          $size      The image size.
+ * @param string          $extension The image extension.
+ *
+ * @since n.e.x.t
+ *
+ * @return string File name with a different extension.
```

Comment:
```suggestion
 * @since n.e.x.t
 *
 * @param WP_Image_Editor $editor    The image editor that handle the image manipulation.
 * @param string          $file      Full path to the image file.
 * @param string          $size      The image size.
 * @param string          $extension The image extension.
 * @return string File name with a different extension.
```